ui/packages/catalog-ui-search/src/main/webapp/component/select/select.collection.view.js

```diff
@@ -20,6 +20,7 @@ const $ = require('jquery')
 const childView = require('./select.view')
 const CustomElements = require('../../js/CustomElements.js')
 const Common = require('../../js/Common.js')
+import { matchesFilter, getAppropriateString } from './filterHelper'
```

Comment:
🤔I think this is big improvement from before, but I think there is still some usability issues , especially when there are many attribute names that contain delimited words that start with the same letters.

My use case is searching by metacard id. Even though this ticket reduced the number of options that I see when I type "id", I still must scroll down the list to select "id", because there are many options that come before "id" alphabetically that contain the word "id" or that start with "id". I do think that it is user-friendly to alphabetize the options in the drop-down, but there could be an easier way to allow a user to search by an attribute by typing in the full name of the attribute.